src/bridge/cockpithttpstream.c

```diff
@@ -418,23 +417,14 @@ relay_headers (CockpitHttpStream *self,
 
   if (self->headers_inline)
     {
-      message = cockpit_json_write_bytes (object);
+      g_autoptr(GBytes) message = cockpit_json_write_bytes (object);
       cockpit_channel_send (channel, message, TRUE);
-      g_bytes_unref (message);
     }
   else
     {
       cockpit_channel_control (channel, "response", object);
     }
 
-  json_object_unref (object);
```

Comment:
duMM question: `json_object_unref`, `g_bytes_unref` and `g_has_table_unref` are the `free` methods for `JsonObject`, `GBytes` and `GHashTable` respectively. So if using `g_autoptr` it is smart enough to know how to properly free these things up?

Reply:
neverMindMe: https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#g-autoptr :)